x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/translations.ts

```diff
@@ -52,10 +52,13 @@ export const CONDITION_FIELD_DESCRIPTION: { [K in ConditionEntryField]: string }
   ),
 };
 
-export const OPERATOR_TITLE: { [K in ConditionEntry['operator']]: string } = {
-  included: i18n.translate('xpack.securitySolution.trustedapps.card.operator.includes', {
+export const OPERATOR_TITLES: { [K in ConditionEntry['operator']]: string } = {
+  included: i18n.translate('xpack.securitySolution.trustedapps.card.operator.is', {
     defaultMessage: 'is',
   }),
+  wildcard_caseless: i18n.translate('xpack.securitySolution.trustedapps.card.operator.matches', {
```

Comment:
Nit: It would be a bit better to derive the keys from the `OperatorEntryField` instead of hardcoding them:
```
export const OPERATOR_TITLES: { [K in OperatorEntryField]: string } = {
  [OperatorEntryField.included]: i18n.translate('xpack.securitySolution.trustedapps.card.operator.is', {
    defaultMessage: 'is',
  }),
  [OperatorEntryField.wildcard_caseless]: i18n.translate('xpack.securitySolution.trustedapps.card.operator.matches', {
    defaultMessage: 'matches',
  }),
};
```

Reply:
Quite right 👍